bootcamp/tutorials/quickstart/hybrid_demo.ipynb

```diff
@@ -0,0 +1,584 @@
+{
```

Comment:
<p>Add links for the following concepts:</p><ul><li>[Milvus Hybrid Search](<a href="https://milvus.io/docs/multi-vector-search.md" target="_blank">https://milvus.io/docs/multi-vector-search.md</a>)</li><li>BGE-M3 model</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1389/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>